src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketsTelemetry.cs

```diff
@@ -71,5 +85,58 @@ public void ConnectFailedAndStop(SocketError error, string? exceptionMessage)
             ConnectFailed(error, exceptionMessage);
             ConnectStop();
         }
+
+        [NonEvent]
+        public void BytesReceived(long count)
+        {
+            Interlocked.Add(ref _bytesReceived, count);
+        }
+
+        [NonEvent]
+        public void BytesSent(long count)
```

Comment:
Why are these arguments long rather than int? It makes sense for the fields to be 64-bit, but individual send/receive operations are typed to never transfer more than 32-bits worth of data, no?

Reply:
You are right, they can be int32. Fixed.